homework.md

```diff
@@ -44,70 +44,72 @@ Using CSS, how would I select every `<h1>` tag on the page?
 
 What is JavaScript?
 
-A programming language used on the internet
+JavaScript is a programming language used in web development. It gives web pages interactive elements that engage a user.
 
 ## Question 8
 
 What three problem does Git & Github solve?
 
-When people want to show off code to each other they can put it on Github
+- Git is a version control system that lets you manage and keep track of your source code history.
+- GitHub is a cloud-based hosting service that lets manage Git repositories.
+- It allows for collaboration without compromising the integrity of the original project.
 
 ## Question 9
 
 What happens when you `fork` a repository?
 
-You delete it
+A fork is a copy of a repository. It allows you to freely experiment with changes without affecting the original project.
 
 ## Question 10 
 
 What happens when you clone a repostory?
 
-It send it to a friend
+when you clone a repo, you make a copy of the complete history.
 
 ## Question 11
 
 What is a Pull Request?
 
-When you send a file over the internet
+A pull request is you are requesting the official repo owner to pull some changes from your own repo.
 
 ## Question 12
 
 What is the Git History log?
 
-It lets you see into the future
+The git log command shows a list of all the commits made to a repository.
 
 ## Question 13
 
 What does it meant to "push" changes to Github?
 
-You send the changes of your local repository to Github
+When you push changes, you send the saved changes in your local repository to the remote repository on GitHub.
 
 ## Question 14
 
 What is Github?
 
-A place for us to store and share our code.
+GitHub is a Git repository hosting service, but it adds many of its own features.
 
 ## Question 15
 
 What does it mean to "Commit your changes"?
 
-Send you code to Github
+The commit command is used to save your changes to the local repository.
 
 ## Question 16
 
 What program do we use to write code at CodeYourFuture?
 
-Microsoft Word
+HTML, CSS, JavaScript.
```

Comment:
I think HTML, CSS and JavaScript are not programs.  Can you please check this answer again?